src/components/board/Card.jsx

```diff
@@ -1,7 +1,52 @@
 import React from "react";
+import Image from "next/image";
+import Judge from "../../../public/images/judge.png";
+import { GrMail } from "react-icons/gr";
 
-const Card = () => {
-  return <div>Card</div>;
+const Card = (props) => {
+  const { name, title, year, major, pronouns, linkedIn, email } = props;
+
+  return (
+    <div className="cardContainer font-poppins m-4 bg-white w-[265px] h-[400px] rounded-3xl shadow flex flex-col items-center justify-center">
+      <div className="contentContainer w-56 h-5/6 flex flex-col justify-center items-start">
+        <div className="photoContainer flex w-full h-[198px]">
+          <Image
+            src={Judge}
+            width={500}
```

Comment:
width={500} and height={500} are in pixels. Try opening a `classname="..."` here and use fractions or percentages